### ISSUE TITLE
CSS used in docs is different than distributed version

### ISSUE THREAD

**Issue by blacktm** [18F/web-design-standards#1590]
## Description

The [documentation site](https://standards.usa.gov) seems to use a different version of the standards than what is currently released and [available for download](https://standards.usa.gov/download/). The effect is that the examples on `standards.usa.gov` (like a [basic header](https://standards.usa.gov/headers/basic/)) are rendered differently using its `styleguide.css` than when using the official `uswds.css` release.

## Steps to reproduce the issue

1. Download the [latest version](https://standards.usa.gov/download/).
2. Enter the `uswds-0.13.2` directory and create an `index.html` file containing the code for the [documentation page template](https://standards.usa.gov/page-templates/#documentation-page).
3. Update the `/assets` URLs to point to the USWDS files, for example, `src="/assets/img/icon-dot-gov.svg"` would become `src="img/icon-dot-gov.svg"`
4. Try alternating between these two CSS files:
```html
<link rel="stylesheet" href="css/uswds.css">
  and...
<link rel="stylesheet" href="https://standards.usa.gov/assets/css/styleguide.css">
```

**Comment by juliaelman** [comment]
@blacktm thanks for opening the issue. A few questions for you, which are more for our user knowledge base than debugging:

1. Did you copy the source code provided for the page templates?
1. Are you currently using the CSS or Sass for styling your project?

The issue is around how the `$em-base` variable is set/imported on the Standards from Bourbon. We are currently "pinned" to `Bourbon 4.2.7` which sets the [`!default`](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_defaults_) value for the `$em-base` var [here](https://github.com/thoughtbot/bourbon/blob/v4.2.7/app/assets/stylesheets/settings/_px-to-em.scss). When we [changed the the variables to all have `!default`](https://github.com/18F/web-design-standards/commit/9798a5216580c8a7bf73c7b68d07ff7a62658c16#diff-455e12140daafe4506437850181e1041) this worked for some of the variables, but not for the `$em-base`, since the default is already declared in Bourbon.

**Comment by blacktm** [comment]
@juliaelman Yes, I [downloaded the package](https://standards.usa.gov/download/) from the site and used the CSS included. I then browsed through the header and footer components in the documentation and copied the code provided in the accordions. I noticed the inconsistencies when comparing the online demos, for example the [basic header](https://standards.usa.gov/headers/basic/), with what I was getting when linking to the CSS from the downloaded package.

Hope that helps!
